src/topic_training/src/Begin_Here.md

```diff
@@ -0,0 +1,122 @@
+# WRoverPlayground - Topic Training
+Now that you have completed the Tools Training, you are ready to learn about ROS topics. From the Setup Training, you should be familiar with:
+- the basics of ROS Nodes
+- rosnode
+- and roscore.
+
+Once you complete this training, you should be familiar with:
+- the purpose of topics
+- the relationship between Publishers and Subscribers
+- setting up Publishers and Subscribers
+- running Publishers and Subscribers
+- rqt_graph
+- rqt_plot
+- custom topic types
+- and rostopic type {topic}.
+
+## The Purpose of Topics
+The word "topic" sounds very general, but it is actually a specific concept that allows ROS nodes to communicate with each other. This is extremely important because one node cannot possibly contain all the information a rover would need.
+
+Topics allow us to specialize nodes, so they can execute their own code/tasks, while also sending and receiving information. For example, a node could receive data and send it to another one, a node could receive and analyze data and have another node do some action with it, a node could receive data from multiple nodes, etc.
```

Comment:
I like the comment about the specialization of nodes.  I would expound on it and additionally emphasize the 'hot-swap'-ability of code using topics.  Since topics don't really care who is publishing or subscribing, it's not only easy to write a node that serves, say, GPS data from a physical sensor, but it is equally easy to create a mock node to test the other nodes without hardware *without changing the other nodes*, since the interface provided through the published data is equivalent.

Reply:
Fixed in f580517.